knowledge-challenge-app/src/Components/LO/__test__/LO.test.js

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import { unmountComponentAtNode } from "react-dom";
+import { BrowserRouter as Router } from "react-router-dom";
+import { render, screen } from "@testing-library/react";
+import "@testing-library/jest-dom";
+import LO from "../LO";
+
+const testCookies = {
+  sessionId: "dca40c1e-df73-441c-8706-174c1d262ce7",
+  userID: "2",
+  email: "zayn68@gmail.com",
+  isAdmin: "0",
+};
+
+let container = null;
+beforeEach(() => {
+  container = document.createElement("div");
+  document.body.appendChild(container);
+});
```

Comment:
this method is not recommended for react. Better to use state / conditional renderings